angular/app/pages/landing/landing.scss

```diff
@@ -42,9 +40,9 @@ body{
 .Landing-getStarted{
     padding: 12px 30px;
     font-weight: bold;
-    color: @black !important;
+    color: $black !important;
     border-radius: 4px;
-    text-transform: box-shadow 250ms;
+    transition: box-shadow 250ms;
```

Comment:
I looked around for text-transform: box-shadow and it does nothing. I thought maybe I was wrong but I looked it up again and there is no "box-shadow" value for text-transform.

All I could guess was that the original designer intended to use transition since there is a 250ms in the line.

See:
* **https://developer.mozilla.org/en-US/docs/Web/CSS/transition**
* https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform
* https://developer.mozilla.org/en-US/docs/Web/CSS/transform